### ISSUE TITLE
import cloudflare_workers_kv documentation is wrong

### ISSUE THREAD

**Issue by antigenius0910** [cloudflare/terraform-provider-cloudflare#1398]
### Confirmation

- [X] My issue isn't already found on the issue tracker.
- [X] I have replicated my issue using the latest version of the provider and it is still present.

### Terraform and Cloudflare provider version

v3.6.0

### Affected resource(s)

cloudflare_workers_kv

### Terraform configuration files

```hcl
I test what you have on terraform registry as example
https://registry.terraform.io/providers/cloudflare/cloudflare/latest/docs/resources/workers_kv


resource "cloudflare_workers_kv_namespace" "example_ns" {
  title = "test-namespace"
}

resource "cloudflare_workers_kv" "example" {
  namespace_id = cloudflare_workers_kv_namespace.example_ns.id
  key = "test-key"
  value = "test value"
}
```

### Debug output

```terraform
ᐅ TF_LOG=DEBUG terraform import cloudflare_workers_kv_namespace.example 895ac02c008443aab205e8939e471bb9_test-key
2022-01-18T10:47:51.958-0600 [DEBUG] Adding temp file log sink: /var/folders/xb/77gkpnrj527cqm82yf4x8_l40000gp/T/terraform-log788947437
2022-01-18T10:47:51.958-0600 [INFO]  Terraform version: 1.0.9
2022-01-18T10:47:51.958-0600 [INFO]  Go runtime version: go1.16.4
2022-01-18T10:47:51.958-0600 [INFO]  CLI args: []string{"/usr/local/bin/terraform", "import", "cloudflare_workers_kv_namespace.example", "895ac02c008443aab205e8939e471bb9_test-key"}
2022-01-18T10:47:51.958-0600 [DEBUG] Attempting to open CLI config file: /Users/yen/.terraformrc
2022-01-18T10:47:51.958-0600 [DEBUG] File doesn't exist, but doesn't need to. Ignoring.
2022-01-18T10:47:51.959-0600 [DEBUG] ignoring non-existing provider search directory terraform.d/plugins
2022-01-18T10:47:51.959-0600 [DEBUG] ignoring non-existing provider search directory /Users/yen/.terraform.d/plugins
2022-01-18T10:47:51.959-0600 [DEBUG] ignoring non-existing provider search directory /Users/yen/Library/Application Support/io.terraform/plugins
2022-01-18T10:47:51.959-0600 [DEBUG] ignoring non-existing provider search directory /Library/Application Support/io.terraform/plugins
2022-01-18T10:47:51.959-0600 [INFO]  CLI command args: []string{"import", "cloudflare_workers_kv_namespace.example", "895ac02c008443aab205e8939e471bb9_test-key"}
Error: resource address "cloudflare_workers_kv_namespace.example" does not exist in the configuration.

Before importing this resource, please create its configuration in the root module. For example:

resource "cloudflare_workers_kv_namespace" "example" {
  # (resource arguments)
}
```

### Expected output

I expect once I provide namespace_id and key for import it should import the state, just like what your tutorial mentioned

![Screen Shot 2022-01-18 at 10 50 17 AM](https://user-images.githubusercontent.com/5915590/149981691-443ae46e-9ea3-4b2e-b057-ae531581743a.png)

below image is from my account
![Screen Shot 2022-01-18 at 10 52 21 AM](https://user-images.githubusercontent.com/5915590/149982037-376b1ab2-82cd-4db7-aac6-659ec2a932f0.png)

```terraform
ᐅ terraform import cloudflare_workers_kv_namespace.example 895ac02c008443aab205e8939e471bb9_test-key 

some import result...
```

### Actual output

```terraform
ᐅ terraform import cloudflare_workers_kv_namespace.example 895ac02c008443aab205e8939e471bb9_test-key 

Error: resource address "cloudflare_workers_kv_namespace.example" does not exist in the configuration.

Before importing this resource, please create its configuration in the root module. For example:

resource "cloudflare_workers_kv_namespace" "example" {
  # (resource arguments)
}
```
or
```terraform
ᐅ terraform import cloudflare_workers_kv_namespace.example_ns 895ac02c008443aab205e8939e471bb9_test-key

cloudflare_workers_kv_namespace.example_ns: Importing from ID "895ac02c008443aab205e8939e471bb9_test-key"...
cloudflare_workers_kv_namespace.example_ns: Import prepared!
  Prepared cloudflare_workers_kv_namespace for import
cloudflare_workers_kv_namespace.example_ns: Refreshing state... [id=895ac02c008443aab205e8939e471bb9_test-key]
╷
│ Error: Cannot import non-existent remote object
│ 
│ While attempting to import an existing object to
│ "cloudflare_workers_kv_namespace.example_ns", the provider detected that no
│ object exists with the given id. Only pre-existing objects can be imported; check
│ that the id is correct and that it is associated with the provider's configured
│ region or endpoint, or use "terraform apply" to create a new remote object for
│ this resource.
```

or 
```terraform
ᐅ terraform import cloudflare_workers_kv.example 895ac02c008443aab205e8939e471bb9_test-key 
cloudflare_workers_kv.example: Importing from ID "895ac02c008443aab205e8939e471bb9_test-key"...
╷
│ Error: Plugin did not respond
│ 
│ The plugin encountered an error, and failed to respond to the
│ plugin.(*GRPCProvider).ImportResourceState call. The plugin logs may contain more
│ details.
╵


Stack trace from the terraform-provider-cloudflare_v3.6.0 plugin:

panic: runtime error: index out of range [1] with length 1

goroutine 50 [running]:
github.com/cloudflare/terraform-provider-cloudflare/cloudflare.parseId(...)
        github.com/cloudflare/terraform-provider-cloudflare/cloudflare/resource_cloudflare_worker_kv.go:90
github.com/cloudflare/terraform-provider-cloudflare/cloudflare.resourceCloudflareWorkersKVImport(0xc0001bfb80, {0x1910e40, 0xc0000c6780})
        github.com/cloudflare/terraform-provider-cloudflare/cloudflare/resource_cloudflare_worker_kv.go:78 +0x1c9
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Provider).ImportState(0xc00009c720, {0x1a42878, 0xc000310500}, 0xc000121a28, {0xc00030c0c0, 0x29})
        github.com/hashicorp/terraform-plugin-sdk/v2@v2.10.1/helper/schema/provider.go:384 +0x142
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*GRPCProviderServer).ImportResourceState(0xc0000d00f0, {0x1a42878, 0xc000310500}, 0xc00061a2c0)
        github.com/hashicorp/terraform-plugin-sdk/v2@v2.10.1/helper/schema/grpc_provider.go:1043 +0xdf
github.com/hashicorp/terraform-plugin-go/tfprotov5/tf5server.(*server).ImportResourceState(0xc0000da800, {0x1a42920, 0xc000618420}, 0xc00074e0a0)
        github.com/hashicorp/terraform-plugin-go@v0.5.0/tfprotov5/tf5server/server.go:628 +0x262
github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5._Provider_ImportResourceState_Handler({0x18e5b00, 0xc0000da800}, {0x1a42920, 0xc000618420}, 0xc000576120, 0x0)
        github.com/hashicorp/terraform-plugin-go@v0.5.0/tfprotov5/internal/tfplugin5/tfplugin5_grpc.pb.go:398 +0x170
google.golang.org/grpc.(*Server).processUnaryRPC(0xc0000c48c0, {0x1a50ea8, 0xc000404480}, 0xc0000fc000, 0xc00044f320, 0x1f82418, 0x0)
        google.golang.org/grpc@v1.40.0/server.go:1297 +0xccf
google.golang.org/grpc.(*Server).handleStream(0xc0000c48c0, {0x1a50ea8, 0xc000404480}, 0xc0000fc000, 0x0)
        google.golang.org/grpc@v1.40.0/server.go:1626 +0xa2a
google.golang.org/grpc.(*Server).serveStreams.func1.2()
        google.golang.org/grpc@v1.40.0/server.go:941 +0x98
created by google.golang.org/grpc.(*Server).serveStreams.func1
        google.golang.org/grpc@v1.40.0/server.go:939 +0x294

Error: The terraform-provider-cloudflare_v3.6.0 plugin crashed!

This is always indicative of a bug within the plugin. It would be immensely
helpful if you could report the crash with the plugin's maintainers so that it
can be fixed. The output above should help diagnose the issue.
```

### Steps to reproduce

follow your tutorial https://registry.terraform.io/providers/cloudflare/cloudflare/latest/docs/resources/workers_kv
```terraform
ᐅ cat test.tf 
resource "cloudflare_workers_kv_namespace" "example_ns" {
  title = "test-namespace"
}

resource "cloudflare_workers_kv" "example" {
  namespace_id = cloudflare_workers_kv_namespace.example_ns.id
  key = "test-key"
  value = "test value"
}
```

```terraform
ᐅ terraform apply

Terraform used the selected providers to generate the following execution plan.
Resource actions are indicated with the following symbols:
  + create

Terraform will perform the following actions:

  # cloudflare_workers_kv.example will be created
  + resource "cloudflare_workers_kv" "example" {
      + id           = (known after apply)
      + key          = "test-key"
      + namespace_id = (known after apply)
      + value        = "test value"
    }

  # cloudflare_workers_kv_namespace.example_ns will be created
  + resource "cloudflare_workers_kv_namespace" "example_ns" {
      + id    = (known after apply)
      + title = "test-namespace"
    }

Plan: 2 to add, 0 to change, 0 to destroy.

Do you want to perform these actions?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: yes

cloudflare_workers_kv_namespace.example_ns: Creating...
cloudflare_workers_kv_namespace.example_ns: Creation complete after 0s [id=fdc7a748d6464e5d8665de43a1190044]
cloudflare_workers_kv.example: Creating...
cloudflare_workers_kv.example: Creation complete after 1s [id=fdc7a748d6464e5d8665de43a1190044/test-key]

Apply complete! Resources: 2 added, 0 changed, 0 destroyed.
```

```terraform 
ᐅ cat terraform.tfstate
{
  "version": 4,
  "terraform_version": "1.0.9",
  "serial": 6,
  "lineage": "14d33f42-8f8e-7a44-92a4-e5d4e8315096",
  "outputs": {},
  "resources": [
    {
      "mode": "managed",
      "type": "cloudflare_workers_kv",
      "name": "example",
      "provider": "provider[\"registry.terraform.io/cloudflare/cloudflare\"]",
      "instances": [
        {
          "schema_version": 0,
          "attributes": {
            "id": "fdc7a748d6464e5d8665de43a1190044/test-key",
            "key": "test-key",
            "namespace_id": "fdc7a748d6464e5d8665de43a1190044",
            "value": "test value"
          },
          "sensitive_attributes": [],
          "private": "bnVsbA==",
          "dependencies": [
            "cloudflare_workers_kv_namespace.example_ns"
          ]
        }
      ]
    },
    {
      "mode": "managed",
      "type": "cloudflare_workers_kv_namespace",
      "name": "example_ns",
      "provider": "provider[\"registry.terraform.io/cloudflare/cloudflare\"]",
      "instances": [
        {
          "schema_version": 0,
          "attributes": {
            "id": "fdc7a748d6464e5d8665de43a1190044",
            "title": "test-namespace"
          },
          "sensitive_attributes": [],
          "private": "bnVsbA=="
        }
      ]
    }
  ]
}
```

delete the state file and import with 
```terraform
ᐅ terraform import cloudflare_workers_kv_namespace.example fdc7a748d6464e5d8665de43a1190044_test-key
Error: resource address "cloudflare_workers_kv_namespace.example" does not exist in the configuration.

Before importing this resource, please create its configuration in the root module. For example:

resource "cloudflare_workers_kv_namespace" "example" {
  # (resource arguments)
}
```

correct way to import 
```terraform
ᐅ terraform import cloudflare_workers_kv.example fdc7a748d6464e5d8665de43a1190044/test-key
cloudflare_workers_kv.example: Importing from ID "fdc7a748d6464e5d8665de43a1190044/test-key"...
cloudflare_workers_kv.example: Import prepared!
  Prepared cloudflare_workers_kv for import
cloudflare_workers_kv.example: Refreshing state... [id=fdc7a748d6464e5d8665de43a1190044/test-key]

Import successful!

The resources that were imported are shown above. These resources are now in
your Terraform state and will henceforth be managed by Terraform.
```

**Comment by jacobbednarz** [comment]
hi @antigenius0910, is there anything stopping you from sending a PR including the fixes you need here?

**Comment by jacobbednarz** [comment]
fixed by #1595
